src/everest/config/sampler_config.py

```diff
@@ -37,9 +37,12 @@ class SamplerConfig(BaseModel):  # type: ignore
         description="""Whether to share perturbations between realizations.
 """,
     )
+    model_config = ConfigDict(
+        extra="forbid",
```

Comment:
In `extra="forbid",` is the comma `,` really needed? Adding it forces ruff formatting to put everything on a new line.